locale/en-us/found_shop.dialog

```diff
@@ -0,0 +1 @@
+found shop {{name}} work hours are {{hours}} you can find this store on {{location}}
```

Comment:
I think adding punctuation here might help with TTS inflection, i.e.:
`I found shop {{name}} with hours of {{hours}}. You can find this store on {{location}}.`